### ISSUE TITLE
Nuke TWR (Trac #257)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#257]
Kill it. Kill it with fire.

Migrated from https://code.icecube.wisc.edu/ticket/257
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:22:56", 
    "description": "Kill it. Kill it with fire.", 
    "reporter": "nega", 
    "cc": "blaufuss, olivas", 
    "resolution": "fixed", 
    "_ts": "1547241776942948", 
    "component": "combo core", 
    "summary": "Nuke TWR", 
    "priority": "normal", 
    "keywords": "twr", 
    "time": "2011-05-11T20:21:18", 
    "milestone": "", 
    "owner": "nwhitehorn", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20110511T20:24:25: nega changed version from "" to "trunk"

**Comment by icecube-trac** [comment]
Trac update at 20110511T20:44:41: 
* nega changed cc from "" to "blaufuss, olivas"
* nega changed owner from "blaufuss" to ""

**Comment by icecube-trac** [comment]
Trac update at 20110709T20:17:05: nega commented:

> Status?

**Comment by icecube-trac** [comment]
Trac update at 20110712T21:51:28: 
* blaufuss commented:

> only remaining TWR items in  offline-software:
> 
> - TWR specific data classes (physics, status, calibration, geometry)
> - I3DB services that fill above classes.
> 
> We should make a long range plan for eventually dropping these from GCDP frames completely once there is no more TWR analyses going on.
> 
> 


* blaufuss changed resolution from "" to "fixed"
* blaufuss changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20141123T03:37:57: 
* olivas commented:

> Milestone "Happy Time!" version of offline-software deleted


* olivas changed milestone from ""Happy Time!" version of offline-software" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190111T19:10:52: 
* nwhitehorn commented:

> In changeset 168015:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168015"
> remove 'using namespace std' from I3Tray.h and I3Module.h
> offline-software icerec builds and tests clean w/ these patches applied
> closes #257 / #IT257
> ```


* nwhitehorn changed owner from "" to "nwhitehorn"

**Comment by icecube-trac** [comment]
Trac update at 20190111T19:37:09: nwhitehorn commented:

> In changeset 168153:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168153"
> remove 'using namespace std' from I3Tray.h and I3Module.h
> offline-software icerec builds and tests clean w/ these patches applied
> closes #257 / #IT257
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:22:56: nwhitehorn commented:

> In changeset 168951:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168951"
> remove 'using namespace std' from I3Tray.h and I3Module.h
> offline-software icerec builds and tests clean w/ these patches applied
> closes #257 / #IT257
> ```
